design-documents/Payment-Request-API.md

```diff
@@ -0,0 +1,66 @@
+## Overview
+The Payment Request API (PR API) is a [W3C standard](https://www.w3.org/TR/payment-request/) candidate that is meant
+to eliminate checkout forms. It improves user workflow during the purchase process, providing a more consistent user
+experience and enabling merchants to easily leverage different payment methods.
+
+The Payment Request API is designed to be vendor-agnostic, meaning it does not require the use of a particular payment system.
+It's not a new payment method, nor does it integrate directly with payment processors; rather, it is a conduit from
+the user's payment and shipping information to merchants, with the following goals:
+
+* Let the browser act as an intermediary among merchants, users, and payment methods
+* Standardize the payment communication flow as much as possible
+* Seamlessly support different secure payment methods
+* Work on any browser, device, or platform - mobile or otherwise
+* Is an open and cross-browser standard
+
+PR API supports different payment methods like:
+
+* Basic Card (standardized by [W3С](https://w3c.github.io/payment-method-id/#registry)) - a payment method that provides credit,
+debit and prepaid card payment information
+* URL-based (like Google Pay, AliPay, Apple Pay, Samsung Pay, etc.) - anyone can develop and provide their own solution
+
+## General Transaction Flow
+In general, the payment transaction flow looks like this:
+![PR API](http://res.cloudinary.com/dxaxolwzl/image/upload/v1527778787/PR_API_General_Flow.png)
+1. JS components render a payment page and initialize Payment Request API.
+1. The browser performs PR API initialization with specified payment details.
```

Comment:
> with specified payment details.

I'd recommend re-wording this for clarity. AFAIK, the initialization is happening with the list of available payment methods in the Magento store.

With the current wording, it sounds like the implementation would be passing user selections to the PaymentRequest constructor, which I don't believe would be accurate in our implementation (the user chooses the payment type from the PR UI in the browser chrome).